vagrant/__init__.py

```diff
@@ -164,7 +164,7 @@ class Vagrant(object):
         'ubuntu-precise64': 'http://files.vagrantup.com/precise64.box',
     }
 
-    def __init__(self, root=None, quiet_stdout=True, quiet_stderr=True):
+    def __init__(self, root=None, quiet_stdout=True, quiet_stderr=True, env=None):
```

Comment:
I wonder if `env=None` is very useful to add. Maybe we can just let people fiddle with `v.env` directly.